dropwizard-lifecycle/src/main/java/io/dropwizard/lifecycle/setup/ScheduledExecutorServiceBuilder.java

```diff
@@ -1,59 +1,73 @@
 package io.dropwizard.lifecycle.setup;
 
-import com.google.common.util.concurrent.ThreadFactoryBuilder;
-import io.dropwizard.lifecycle.ExecutorServiceManager;
-import io.dropwizard.util.Duration;
-
 import java.util.concurrent.RejectedExecutionHandler;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.ThreadPoolExecutor;
 
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+
+import io.dropwizard.lifecycle.ExecutorServiceManager;
+import io.dropwizard.util.Duration;
+
 public class ScheduledExecutorServiceBuilder {
+
     private final LifecycleEnvironment environment;
     private final String nameFormat;
     private int poolSize;
     private ThreadFactory threadFactory;
     private Duration shutdownTime;
     private RejectedExecutionHandler handler;
+    private Boolean removeOnCancel;
 
-    public ScheduledExecutorServiceBuilder(LifecycleEnvironment environment, String nameFormat, ThreadFactory factory) {
+    public ScheduledExecutorServiceBuilder(final LifecycleEnvironment environment, final String nameFormat, final ThreadFactory factory) {
```

Comment:
Technically, constructor and method parameters are local to the code block and if they are immutable, should be declared as final by your logic. However, it is your code base and I'll undo the automated IDE fixes.